subsys/bluetooth/controller/Kconfig

```diff
@@ -247,6 +247,80 @@ config BT_CTLR_ADVANCED_FEATURES
 menu "Advanced features"
 	visible if BT_CTLR_ADVANCED_FEATURES
 
+choice
+	prompt "Tx Power"
+	default BT_CTLR_TX_PWR_0
+
+config BT_CTLR_TX_PWR_PLUS_8
+	prompt "+8 dBm"
+	depends on SOC_NRF52840
+	bool
```

Comment:
Subjective, so just a note in case you've missed it:

This can be shortened to

```
config BT_CTLR_TX_PWR_PLUS_8
	bool "+8 dBm"
	depends on SOC_NRF52840
```

`type "prompt text"` works as a shorthand for giving the type and prompt in one go.

Reply:
done.